rds/aurora/mysql/cluster/Acornfile

```diff
@@ -114,7 +120,8 @@ if args.username != "" {
 }
 
 images: user: containerBuild: {
-	context: ".."
+	context: "../../../"
+  dockerfile: "../../../mysql.Dockerfile"
```

Comment:
@sosodev The indentation looks a bit odd here. Have you tried running `acorn fmt` on this file?

Reply:
I'll be honest and say I didn't even realize the cli had a formatter built in. 😂 

Fixed in the latest commit.